aggregation/aggregation_functions.md

```diff
@@ -97,22 +97,22 @@ We're going to write up a fairly complicated query this time. This query compute
 Our query will look like this:
 
 ```sql
-SELECT Post.avgRating, Pre.avgRating
+SELECT Post.avgRating - Pre.avgRating
```

Comment:
I'm not sure what is going on here, I believe this should be a comma. Could you explain why you are changing this to a hyphen?